Source/WebCore/css/CSSFontFace.cpp

```diff
@@ -52,10 +52,15 @@ namespace WebCore {
 
 DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(CSSFontFace);
 
-template<typename T> void iterateClients(HashSet<CSSFontFace::Client*>& clients, T callback)
+template<typename T> void iterateClients(WeakHashSet<CSSFontFace::Client>& clients, T callback)
 {
-    for (auto& client : copyToVectorOf<RefPtr<CSSFontFace::Client>>(clients))
-        callback(*client);
+    Vector<Ref<CSSFontFace::Client>> clientsCopy;
```

Comment:
Any reason you chose not to use WeakHashSet::forEach() which does the copy for you? Is it because you really want to Ref the clients?

Reply:
If you really want to Ref, maybe `copyToVectorOf<Ref<CSSFontFace::Client>>(clients)` would work and would be more concise.

Reply:
`copyToVectorOf` doesn't work because it WeakHashSet doesn't have size().

Reply:
I guess I can add a template specialization for it.

Reply:
Oh... actually, copyToVectorOf works out of box. I just had to change the type to Ref<CSSFontFace::Client> instead of RefPtr<CSSFontFace::Client>.